js/script.js

```diff
@@ -123,6 +127,17 @@ const createTaskCard = (task) => {
     taskCard.appendChild(deleteButton);
   }
 
+  if (!task.isCompleted) {
+    const checkButton = document.createElement("button");
+    checkButton.className = "btn-check";
+    checkButton.textContent = "✔️";
+    checkButton.addEventListener("click", () => {
+      task.isCompleted = true;
+      renderTasks();
+    });
+    taskCard.appendChild(checkButton);
```

Comment:
```suggestion
    markAsDone();
```

Move it to a separate function